wurst/systems/modes/Respawn.wurst

```diff
@@ -1,16 +1,19 @@
 package Respawn
 import RegisterEvents
 import TrollExtensions
+import TribeUtils
 import TimerExtensions
 import ClosureTimers
 import GameConfig
 import TimerUtils
 import TimerDialog
+import ChatCommands
+import LinkedList
 
 @configurable constant GRACE_PERIOD_REVIVE_DELAY = 10.0
 
 function isGracePeriodActive() returns boolean
-    return udg_NOOB_TIME
+    return gameConfig.isGracePeriodEnabled() and udg_NOOB_TIME
```

Comment:
Are these not redundant?

Reply:
No, one refers to the modes of the game and the other refers to whether noob time is still active

Reply:
Yes, but grace period is not enabled, isn't `udg_NOOB_TIME` always false?

Reply:
Actually `udg_NOOB_TIME` is default `true` so hence the check